portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -1199,7 +1199,7 @@ definition {
 
 	macro viewOptionSingleSelection {
 		AssertElementPresent(
-			key_bodyText = "${bodyText}",
+			key_optionName = "${optionName}",
 			key_fieldName = "${fieldName}",
```

Comment:
@gleicelisbino, can you please sort these parameters alphabetically?

Reply:
@natocesarrego Okay! I will do this!